memgraph_versioned_docs/version-2.2.1/how-to-guides/streams/kafka/implement-transformation-module.md

```diff
@@ -192,7 +192,6 @@ So let's create `c_transformation.cpp` and start to populate it!
 ```cpp
 #include <exception>
 #include <string>
-#include <string_view>
```

Comment:
I think these changes should be primarily done on the current version (in the `docs` folder) and than copy back to every version up-to 2.0.0 where the MGP API was rewamped.

Reply:
Will do.

Reply:
Benjo is correct, you only need to apply the changes in the `/docs` directory, and the DX team will propagate them where required. Usually, we only apply it to the latest version (currently 2.3.0), but if it's only minor changes (copy/paste friendly), then we go even further down the versions.

Reply:
I originally miss-understood, my bad. It should be correct now, only docs/how-to-guides/streams/kafka/implement-transformation-module.md should have the changes.